### DISCUSSION TITLE
error con las librerias despues de instalar gdb

### DISCUSSION THREAD

**Discussion by sbarriag** [IIC2133-PUC/2024-2#84] [Tarea 0] [answered]
Hola, estaba haciendo el setup de gdb para poder ver errores en mi t0 y cuando termine de crear la carpeta .vscode y abro el main.c me sale esto

![image](https://github.com/user-attachments/assets/5e692316-7578-4d91-9285-26f25a462fba)

no se si se deberá a que hice algo mal o en la creacion de la carpeta .vscode o en el .json que va dentro, la carpeta la cree en el repo así 

![image](https://github.com/user-attachments/assets/b1b1f73c-01e6-43bb-be53-10d2332f512c)

y el contenido de mi .json es este

![image](https://github.com/user-attachments/assets/c0b42fe0-d0ab-4f24-ab35-1fc4bcf01670)

ojala me puedan ayudar 
muchas gracias!

**Comment by sebacarrascop** [comment, marked as answer]
Hola! 
Como ayudantía no hemos recomendado el uso de algún debugger como gdb, por lo que no contamos con el conocimiento para ayudarte.
Dada la dificultad de las tareas puedes realizar "debugs" con simples prints en consola, si tienes problemas con gdb te recomiendo realizar eso o seguir las recomendaciones que te entrega vscode para solucionar el problema.
Saludos.
